File: program357.cpp
/*
Enter the number :
5
*       1       *       2       *       3       *       4       *

*/
#include<iostream>
using namespace std;

class Pattern
{
    private:
        int iNo1;

    public:
        Pattern(int a)
        {
            this->iNo1 = a;
            
        }

        void DisplayPattern()
        {
            int iCnt=0;
			for(iCnt=1;iCnt<=iNo1;iCnt++)
			{
				cout<<"*\t"<<iCnt<<"\t";
			}
		
        }
};

int main()
{
	int iValue1=0;
	
	cout<<"Enter the number :"<<endl;
	cin>>iValue1;
	
	
	
    Pattern obj(iValue1);
    obj.DisplayPattern();

    return 0;
}

File: program357.java
import java.lang.*;
import java.util.*;

class program357
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);
		
		System.out.println("Enter the number:");
		int iNo=sobj.nextInt();
		
		int iMask=0x00000001;
		int iResult=0;
		int iCnt=0;
		
		for(int i=1;i<=32;i++)
		{
			iResult=iNo&iMask;
			
			if(iResult==iMask)
			{
				iCnt++;
			}
			iMask=iMask<<1;
		}
		
		System.out.println("Number if ON bits are:"+iCnt);
	}
}